lib/github/middleware.js

```diff
@@ -41,9 +41,13 @@ module.exports = function middleware (callback) {
     enhanceOctokit(context.github, context.log)
 
     newrelic.setControllerName(`github/events.${context.name}`)
+    let webhookEvent = context.name
+    if (context.payload.action !== undefined) {
```

Comment:
I think this should be `!= null` in case the key exists in the dictionary but has a null value.

Reply:
I just updated to check for truthiness instead.